engine/jsonlint/reporter.js

```diff
@@ -0,0 +1,79 @@
+const template = module.exports = {
+    run: function(onFinish) {
+        process.stdin.resume();
+        process.stdin.setEncoding("utf8");
+
+        let lingeringLine = "";
+        let lines = [];
+        
+        process.stdin.on("data", function(chunk) {
+            let linesParsed = chunk.split('\n');
+            linesParsed[0] = lingeringLine + linesParsed[0];
+            lingeringLine = linesParsed.pop();
+            linesParsed.forEach((line) => {
+                if (line !== null) {
+                    lines.push(line);
+                }
+            });
+        });
+
+        process.stdin.on("end", function() {
+            onFinish(lines);
+        });
+      
+    },
+};
+
+const reporter = module.exports = {
+    run: function(regex, convertMatch) {
+        let results = [];
+
+         /**
+         * Generates standard model form each line
+         * @param {*} line Line to parse
+         */
+        function processLine(line) {
+            //console.log(line);
```

Comment:
Please do not commit commented code; you may implement 'verbose' or 'dev' mode where such info will be logged.